### PR TITLE
Updated the logging as requested

### PR DESCRIPTION
**Pull request by AryanM1223**

### CODE REVIEW

File: controllers/groupController.js [hunk 1]
```diff
@@ -449,13 +456,17 @@ exports.fetchGroupDetails = async (req, res) => {
...
+    errorLogger.error(`An error occurred while fetching group details for ${groupId}`,
+     error
+     );
+
     res.status(500).json({ error: "Internal Server Error" });
   }
 };
 
 exports.updateGroupDetails = async (req, res) => {
   const { group_id, name, description, type } = req.body;
```
Comment (BharatShukla30) [inline controllers/groupController.js:468]: groupId is coming from here change it here too

File: controllers/groupController.js [hunk 2]
```diff
@@ -485,8 +497,8 @@ exports.updateGroupDetails = async (req, res) => {
 };
 
 exports.updateIcon = async (req, res) => {
-  const { group_id, icon } = req.body;
-  const group = await Group.findById(new ObjectId(group_id));
+  const { groupId, icon } = req.body;
```
Comment (BharatShukla30) [inline controllers/groupController.js:500]: ask lakshit to make change on the frontend too

**Review by BharatShukla30 — CHANGES_REQUESTED** [review]
ask laskhit to make the same changes on frontend API requests also address the comments

**Comment by AryanM1223** [comment]
will do.

**Review by BharatShukla30 — APPROVED** [review]
Good work.

**Pull request merged**